DependencyInjection/Configuration.php

```diff
@@ -20,14 +21,21 @@
  * limitations under the License.
  */
 
-class Configuration
+class Configuration implements ConfigurationInterface
 {
-    public function getConfigTree()
+    private string $alias;
```

Comment:
Typed properties are available only for PHP7.4 (https://www.php.net/manual/en/language.oop5.properties.php#language.oop5.properties.typed-properties), here we allow 7.3 as well I think